projects/packages/videopress/src/class-stats.php

```diff
@@ -160,7 +164,7 @@ public static function prepare_featured_stats( $dates, $total_days ) {
 		foreach ( $dates as $date_info ) {
 			$date_totals = $date_info['total'];
 
-			if ( $counter < floor( $total_days / 2 ) ) {
+			if ( $counter < floor( $period_count / 2 ) ) {
```

Comment:
I thought about this too, I decided on keeping this same function and using it this way because from what I can tell, this won't be a performance issue. The backend only calls one database query per function call because it just calculates which date it needs to go back to. The processing of this data is negligent because it only has to run through 2 items in a for loop since it's two years of data, which is much less than the 60 items in the for loop that it does for the 60 days check before this.  

Because performance should be negligent, I figured it probably wouldn't be worth the time creating a separate function for it. An additional positive is it makes the types simpler on the frontend to not have to have a conditionally present "previous" key. I know you're probably gone on parental leave today but for whoever re-reviews this, let me know what you think 😄